modules/services/holo-admin.nix

```
@@ -0,0 +1,38 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.holo-admin;
+in
+
+{
+  options.services.holo-admin = {
+    enable = mkEnableOption "Holo Admin";
+
+    package = mkOption {
+      default = pkgs.holo-admin;
+      type = types.package;
+    };
+  };
+
+  # TODO: enable 'sudo nixos-rebuild ...' and 'sudo zerotier-cli -j info`
+  # TODO: make "after" the systemd phase that locates the hpos-state.json file
+  # TODO: set the HPOS_STATE_PATH environment variable
+  config = mkIf cfg.enable {
+    environment.systemPackages = with pkgs; [
+      hpos-state-gen-cli # for testing
+    ];
+    systemd.services.holo-admin = {
```

Set up something like this for now:

```suggestion
    systemd.services.holo-admin = {
      environment.HPOS_STATE_PATH = "/tmp/hpos-state.json";
```

I'll fix that up once HPOS Init work is done.